### PR TITLE
Fix translateFragment API when using a documentFragment object

### PR DESCRIPTION
**Pull request by Phyks**
As discussed in https://github.com/fabi1cazenave/webL10n/issues/73.

This is the fix I am using currently. If you have any opinion and possible improvement, please let me know.

### CODE REVIEW

**Comment by Rob--W** [comment]
This change is non-standard. E.g. I know that IE doesn't have a name property on constructors of host objects.

Please amend the PR to use the standard `element.nodeType` as I suggested in the comment on your bug report.

**Comment by Phyks** [comment]
@Rob--W Sorry, missed your comment on the issue. Should be fixed with the latest commit.

File: l10n.js [hunk 1]
```diff
@@ -979,7 +979,9 @@ document.webL10n = (function(window, document, undefined) {
     }
 
     // translate element itself if necessary
-    translateElement(element);
+    if (element.nodeType != 11) {
```
Comment (Rob--W) [inline l10n.js]: Use `.nodeType === 1` instead. Text nodes (nodeType 3), documents (nodeType 9?), comments, etc. don't support getAttribute either.

And please squash your commits (https://help.github.com/articles/about-git-rebase/).

**Comment by Phyks** [comment]
@Rob--W Should be ok by now.

**Comment by Rob--W** [comment]
Thanks for the patch!

**Pull request merged**